### ISSUE TITLE
supabase executable requires `supabase/config.toml`

### ISSUE THREAD

**Issue by vixalien** [supabase/cli#554]
# Bug report

## Describe the bug

when you run the executable from github releases on linux, it throws this error:

```
Error: stat $(pwd)/supabase/config.toml: not a directory
Try rerunning the command with --debug to troubleshoot the error.
```

`$(pwd)` is the current working directory.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to github releases for supabase
2. download latest tar.gz for linux amd64
3. extract the archive
4. run supabase eg. `supabase completion bash`
5. given error is throw

## Expected behavior

working normally

## System information

- OS: arch linux
- Browser does not apply
- Version of supabase: 1.11.0
- Version of Node.js: 18.9.0

## Additional context

creating the toml file is non-trivial too, because the binary is named `supabase` and the config file must exist at the current directory in the path `supabase/config.toml`, so you must first rename the supabase binary and then create the empty config file.

**Comment by sweatybridge** [comment]
Hi @vixalien, `supabase/config.toml` file is created by the command [`supabase init`](https://supabase.com/docs/guides/cli/local-development) to contain project specific customisations. Typically users will run `supabase init` after cloning a git repository.

In the linked commit of this issue, I see that you have added a [prepare step](https://github.com/archlinux/aur/commit/004520a725c6938890f06efde164bbe03bd4d343#diff-c13dbcca92f9ff12cd26ecce958be3f9ee8563baace04f7a463a6d2dd4252e0bR19) to the package manager installation script. I believe it should not be required as users will typically run init for each project.

**Comment by sweatybridge** [comment]
Also thanks for reporting this issue, I will find another way to fix it.

**Comment by vixalien** [comment]
> Also thanks for reporting this issue, I will find another way to fix it.

I believe that's what needed, but we also need to generate type completions, and that even requires the config file

**Comment by sweatybridge** [comment]
Could you give the latest version a try? The bug was a regression from v1.8.2 and should be fixed since v1.11.1.
